company/team/index.md

```diff
@@ -856,3 +856,13 @@ Outside of work, he enjoys golfing, cheering on his favorite LA sports teams and
 - [rafal@sourcegraph.com](mailto:rafal@sourcegraph.com), [LinkedIn](https://www.linkedin.com/in/rleszcz/)
 - Raf lives with his family outside of Poznan (Poland), surrounded by forests, boars, deers, and other wildlife.
 He likes to travel, visit places off the beaten path, and trek in the mountains. Raf also spends bing chunk of his free time playing video games, both recent titles and retro ones. He has ten different gaming consoles, including NES, the first-generation Game Boy, and Commodore 64, all in working condition.
+
+
```

Comment:
```suggestion

```
So sorry!

Reply:
Oh no, where'd this line come from?? 😂 Sorry I'm used to a markdown linter haha.